apps/oxlint/src/tester.rs

```diff
@@ -16,6 +16,11 @@ impl Tester {
     pub fn new() -> Self {
         let cwd = env::current_dir().unwrap();
 
+        // disable multiple workers for diagnostic
+        // because the snapshot could change every time when we are analyzing multiple files
+        // do not unwrap because we can set it only one time.
+        let _ = rayon::ThreadPoolBuilder::new().num_threads(1).build_global();
```

Comment:
I can remove this and put regex back. I thought about mocking the `lint_command_info` to avoid the regex package.
What do you think?

Reply:
This is good.